seshdash/views.py

```diff
@@ -240,3 +254,35 @@ class UserList(generics.ListAPIView):
 class UserDetail(generics.RetrieveAPIView):
     queryset = User.objects.all()
     serializer_class = UserSerializer
+
+
+def get_high_chart_data():
+     context_high_data = {}
```

Comment:
Don't create a new branch everytime the pull request is rejected you need to modify the code do new commits and then send another pull requet with the same branch.

Looks like you're lossing your changes. I don't see any comments here! Refer to my last few rejections please.